### PR TITLE
config: preconfigure mainnet and testnet

### PR DESCRIPTION
**Pull request by bendanzhentan**

### CODE REVIEW

**Review by welkin22 — APPROVED** [review]
LGTM

File: bot.mainnet.toml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+name = "$OPBNB_BRIDGE_BOT_DB_NAME"
+
+[l1-contracts]
+optimism-portal                 = "0x1876EA7702C0ad0C6A2ae6036DE7733edfBca519"
+l2-output-oracle                = "0x153CAB79f4767E2ff862C94aa49573294B13D169"
+l1-cross-domain-messenger       = "0xd95D508f13f7029CCF0fb61984d5dfD11b879c4f"
+
+[l2-standard-bridge-bot]
+contract_address = "$OPBNB_BRIDGE_BOT_L2_STANDARD_BRIDGE_BOT"
+log-filter-block-range = 1000
```
Comment (owen-reorg) [inline bot.mainnet.toml:26]: plz pre-config the whitelist according to https://github.com/bnb-chain/opbnb-bridge-tokens/blob/main/opbnb.tokenlist.json

File: bot.testnet.toml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+name = "$OPBNB_BRIDGE_BOT_DB_NAME"
+
+[l1-contracts]
+optimism-portal                 = "0x4386c8abf2009ac0c263462da568dd9d46e52a31"
+l2-output-oracle                = "0xff2394bb843012562f4349c6632a0ecb92fc8810"
+l1-cross-domain-messenger       = "0xd506952e78eecd5d4424b1990a0c99b1568e7c2c"
+
+[l2-standard-bridge-bot]
+contract-address                = "0x62fa6549e240076D466E1150ce587905c55f12F1"
+log-filter-block-range = 1000
```
Comment (owen-reorg) [inline bot.testnet.toml:27]: plz pre-config the whitelist according to https://github.com/bnb-chain/opbnb-bridge-tokens#opbnb-testnet-token-list

File: bot.testnet.toml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+challenge-time-window = 3
+l2-starting-number = $OPBNB_BRIDGE_BOT_L2_STARTING_NUMBER
+
+[rpcs]
+l1-rpc = "$OPBNB_BRIDGE_BOT_L1_RPC"
+l2-rpc = "$OPBNB_BRIDGE_BOT_L2_RPC"
+
+[tx-signer]
+privkey = "$OPBNB_BRIDGE_BOT_PRIVKEY"
+gas-price = 9000000000
```
Comment (owen-reorg) [inline bot.testnet.toml]:
```suggestion
gas-price = 5000000000
```

File: bot.mainnet.toml [hunk 4]
```diff
@@ -0,0 +1,26 @@
+propose-time-window = 3600
+challenge-time-window = 604800
+
+[rpcs]
+l1-rpc = "$OPBNB_BRIDGE_BOT_L1_RPC"
+l2-rpc = "$OPBNB_BRIDGE_BOT_L2_RPC"
+
+[tx-signer]
+privkey = "$OPBNB_BRIDGE_BOT_PRIVKEY"
+gas-price = 9000000000
```
Comment (owen-reorg) [inline bot.mainnet.toml]:
```suggestion
gas-price = 3000000000
```

File: bot.mainnet.toml (hunk 1 above)
Comment (bendanzhentan) [reply bot.mainnet.toml:26]: Applied all suggestions and force-pushed

**Comment by bendanzhentan** [comment]
Applied all suggestions and force-pushed. Plz take a look @owen-reorg

**Review by owen-reorg — APPROVED** [review]

**Pull request merged**